helpers/score.py

```diff
@@ -0,0 +1,43 @@
+
+from ..database.objects import DBBeatmap, DBScore
+from ..constants.mods import Mods
+
+def calculate_difficulty_multiplier(beatmap: DBBeatmap):
+    return round(
+        (beatmap.hp + beatmap.cs + beatmap.od + min(0, max(16, beatmap.max_combo / beatmap.total_length * 8))) / 38 * 5
+    )
+
+def calculate_mod_multiplier(mods: int):
+    multiplier = 1.00
+    if mods & Mods.Easy:
```

Comment:
I would like to use a different syntax here:
```python
if Mods.Easy in mods
```
This requires the `mods` parameter to be a `Mods` enum.
````python
def calculate_mod_multiplier(mods: Mods)
    …

calculate_mod_multiplier(Mods(score.mods))
```